content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,146 @@
+
+<!--
+TODO describe each of the addons into its own section,
+similar to what we have in https://github.com/readthedocs/addons/blob/main/README.md
+
+Consider also https://about.readthedocs.com/docs-as-code/
+-->
+
+### Analytics
+
+![Analytics Addons](/images/addons-analytics.png)
+
+Analytics allows you to explore your documentation traffic to understand what are the pages your readers read the most.
+With this data, your team is able to **prioritize the work on pages with high traffic** over others with less.
+Besides, you can find out what are the terms your users search the most and
+write documentation pages that include those terms so they can easily find them.
+
+### DocDiff
+
+![Docdiff Addons](/images/addons-docdiff.gif)
+
+DocDiff shows changes made on the rendered versions by **visually highlighing the differences between
+the current pull request and the latest version of the project's documentation**.
+This addons allows reviewers to quickly find what has changed on this pull request
+but also find style/rendering issues before accepting and merging the pull request.
+
+### Pull request notification
+
+![Pull request notification](/images/addons-pull-request-notification.png)
+
+This notification warns users letting them know the documentation page they are reading
+comes from a pull request to clearly distinguish this version from the official one.
+
+
+### Flyout
+
+![Flyout Addons](/images/addons-flyout.png)
+
+We call _flyout_ to the small floating Read the Docs section where all the documentation versions and translations are listed.
+This allows readers to **find the exact version they are looking for** and also read the documention on their own language if it's available.
+Besides, it contains useful links to go to the project's home, builds page and offline formats.
+
+### Non-stable notification
+
+![Non-stable notification](/images/addons-non-stable-notification.png)
+
+A notification on all _non-stable_ version is shown to clearly communicate readers
+they may be reading an outdated version of the documentation.
+This helps readers to avoid confusions and quickly jump into the _latest_ version of the documentation.
+Besides, it **reduces support requests from old version of your product**.
+
+
+### Latest version notification
+
+![Latest notification](/images/addons-latest-notification.png)
+
+A notification shown on _latest_ version to tell readers
+they are reading the _latest/development_ version of the documentation that may include features not yet deployed.
+This will help your readers to avoid confusions on the version they are using and the one they are reading,
+while letting your project to expose these yet unrelease features.
+
+### Search as you type
+
+![Search](/images/addons-search.gif)
+
+Search _as-you-type_ powered by an Elastic backend that allows users to quickly find exactly what they are looking for while typing.
+It also **saves recent searches to have them at the tip of your fingers**.
```

Comment:
```suggestion
It also **saves recent searches to have them at the tip of your fingers**.
```

Is your the docs author or the reader?

Reply:
Hrm, in this case is the reader --which is the one _using_ this feature.